src/Apps/Auction/Routes/__tests__/ConfirmBid.test.tsx

```diff
@@ -0,0 +1,97 @@
+import React from "react"
+import { graphql } from "react-relay"
+
+import { createTestEnv } from "DevTools/createTestEnv"
+
+import { routes_BidQueryResponse } from "__generated__/routes_BidQuery.graphql"
+import { BidQueryResponseFixture } from "Apps/Auction/__fixtures__/routes_BidQuery"
+import { RegisterQueryResponseFixture } from "../../__fixtures__/routes_RegisterQuery"
+import { createBidderPositionSuccessful } from "../__fixtures__/MutationResults/createBidderPosition"
+import { ConfirmBidRouteFragmentContainer } from "../ConfirmBid"
+import { ConfirmBidTestPage } from "./Utils/ConfirmBidTestPage"
+
+jest.unmock("react-relay")
+
+jest.mock("sharify", () => ({
+  data: {
+    APP_URL: "https://example.com",
+  },
+}))
+
+const mockLocation: any = {
+  search: "",
+}
+
+const setupTestEnv = () => {
+  return createTestEnv({
+    TestPage: ConfirmBidTestPage,
+    Component: (props: routes_BidQueryResponse) => (
+      <ConfirmBidRouteFragmentContainer location={mockLocation} {...props} />
+    ),
```

Comment:
This little bit of extra prop injection is necessary because of the way that I included a [location query string toggle for a pricing transparency preview](https://github.com/artsy/reaction/pull/2918/files#diff-732fcbc97d5a2306a5a052ccf1852df4R40). This addition was only intended to clear a path for developers to work on the pricing transparency component + queries, so now it feels a bit contrived to pass an extra `Location` object (as a prop from the router, not `window`- we do use `window.location.assign()` elsewhere currently).

Maybe acceptable in the short term but kind of a pain.